ecs-init/config/common_test.go

```diff
@@ -66,3 +66,13 @@ func TestGetS3BucketMapByRegion(t *testing.T) {
 		})
 	}
 }
+
+func TestCgroupMountOverride(t *testing.T) {
+	cgroupTest := "/ecs/cgroup"
+	os.Setenv("CGROUP_MOUNTPOINT", cgroupTest)
```

Comment:
Any time you're setting an environment variable in a test, you should `defer os.Unsetenv`.